toolbox/githubctl/main.go

```diff
@@ -291,8 +301,10 @@ func readPostsubmitListFromProwConfig(org, repo, branch string) map[string]struc
 
 func init() {
 	flag.Parse()
+	var token string
```

Comment:
Why these changes? Using := to declare is also idiomatic go

Reply:
oh it was due to some previous change when token and error could have been assigned different. I had since removed that code but did not convert this back.